src/compiler/transformers/module/module.ts

```diff
@@ -609,7 +609,10 @@ namespace ts {
         }
 
         function visitImportCallExpression(node: ImportCall): Expression {
-            const argument = visitNode(firstOrUndefined(node.arguments), moduleExpressionElementVisitor);
+            const externalModuleName = getExternalModuleNameLiteral(factory, node, currentSourceFile, host, resolver, compilerOptions);
+            const firstArgument = visitNode(firstOrUndefined(node.arguments), moduleExpressionElementVisitor);
+            // Only use the external module name if it differs from the first argument. This allows us to preserve the quote style of the argument on output.

```

Comment:
If we really wanted to, couldn't we copy the quote style from the first argument string, if present?

Reply:
We don't preserve quote-style on parse, and this is how we handle strings everywhere else (copy from source text if possible).